v0/destinations/active_campaign/transform.js

```diff
@@ -117,10 +128,18 @@ const customTagProcessor = async (message, category, destination) => {
       }
     );
   } catch (err) {
-    throw new CustomError(
-      `Failed to fetch already created tags (${err.response.statusText})`,
-      err.response.status || 400
-    );
+    if (err.response) {
```

Comment:
Rather than repeating this logic, lets create utility fxn for this

Reply:
Done